_data/sites.json

```
@@ -23268,5 +23268,12 @@
         "notes": "Go to the linked page and click \"Удалить учетную запись\".",
         "notes_ru": "Перейдите по ссылке и нажмите кнопку \"Удалить учетную запись\".",
         "domains": ["sdamgia.ru"]
+    },
+    {
+        "name": "Zzo.ai",
+        "url": "https://zzo.ai/profile",
+        "difficulty": "easy",
+        "notes": "Log in, visit the profile page at https://zzo.ai/profile, scroll to the bottom and click the 'Delete Account' button. A warning dialog will appear, click 'Confirm' to delete your account.",
```

```suggestion
        "notes": "Log in, visit the profile page, scroll to the bottom and click the 'Delete Account' button. A warning dialog will appear, click 'Confirm' to delete your account.",
```

Link provided in url tag :)